src/main/java/com/java_essential/kvirikadze/homework/lesson3/issue1/ClassRoom.java

```diff
@@ -0,0 +1,38 @@
+package com.java_essential.kvirikadze.homework.lesson3.issue1;
+
+import static java.lang.String.format;
+
+public class ClassRoom {
+    private Pupil[] pupils = new Pupil[4];
+    private String name;
+
+    public ClassRoom(String name, Pupil pupil1, Pupil pupil2, Pupil pupil3, Pupil pupil4) {
+        this.name = name;
+        this.pupils[0] = pupil1;
+        this.pupils[1] = pupil2;
+        this.pupils[2] = pupil3;
+        this.pupils[3] = pupil4;
+    }
```

Comment:
лучше здесь использовать varargs
```suggestion
    public ClassRoom(String name, Pupil... pupils) {
      
```